src/azure-cli-core/azure/cli/core/tests/test_cloud.py

```diff
@@ -197,12 +166,6 @@ def test_get_known_clouds(self):
 
     def test_modify_known_cloud(self):
         with mock.patch('azure.cli.core.cloud.CLOUD_CONFIG_FILE', tempfile.mkstemp()[1]) as config_file:
-            pool_size = 100
-            p = multiprocessing.Pool(pool_size)
-            p.map(init_known_clouds, [True] * pool_size)
```

Comment:
Did you remove this because it's not needed or because it's flaky?

Reply:
It was all part of https://github.com/Azure/azure-cli/commit/29983162f211ae9fd353268b6ff70161673d8fc3#diff-69bacf5348b997023cd1e8e6f475a0da